### PR TITLE
docs: add neon integration

### PR DESCRIPTION
**Pull request by milasuperstar**
Signed-off-by: Mila Wu <wumila1996@gmail.com>

### CODE REVIEW

File: content/docs/how-to/integrations/neon.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+7. **Password:** copy the **PGPASSWORD** from the `.env` file
+8. **Database:** copy the **PGDATABASE** from the `.env` file
+
+![neon-bytebase-create-instance](/docs/how-to/integrations/neon-bytebase-create-instance.webp)
+
+See [Add an Instance](/docs/get-started/configure-workspace/add-an-instance) for more details.
+
+### Step 3 - Check if the database instance is properly imported
+
+All databases should be sycned properly. Expect some delay if the database instance is large.
```
Comment (adela-bytebase) [inline content/docs/how-to/integrations/neon.md:48]:
```suggestion
All databases should be synced properly. Expect some delay if the database instance is large.
```

Comment (adela-bytebase) [inline content/docs/how-to/integrations/neon.md:48]:
```suggestion
All databases should be synced properly. Expect some delay if the database instance is large.
```

File: content/docs/how-to/integrations/neon.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: Manage Neon Database
+description: Tutorial on how to manage Neon PostgreSQL database
+---
+
+## Overview
+
+[Neon](https://neon.tech/) is a fully managed serverless PostgreSQL that offers cool features such as [database branching](https://neon.tech/docs/introduction/branching/), and bottomless storage.
+
+You can create a branch of your PostgreSQL database on Neon and use Bytebase to manage the database development lifecycle for those databases.
```
Comment (tianzhou) [inline content/docs/how-to/integrations/neon.md:10]:
```suggestion
You can create a PostgreSQL instance on Neon and use Bytebase to manage the database development lifecycle for those databases.
```

**Review by tianzhou — APPROVED** [review]
LGTM. This tutorial is quite clear.

File: content/docs/how-to/integrations/neon.md (hunk 1 above)
Comment (milasuperstar) [reply content/docs/how-to/integrations/neon.md:48]: good eye 👀

**Pull request merged**